hibernate-core/src/main/java/org/hibernate/query/sqm/function/SqmFunctionDescriptor.java

```diff
@@ -36,4 +36,24 @@ Expression generateSqlExpression(
 			SqmToSqlAstConverter converter,
 			SqlAstCreationState creationState);
 
+	/**
+	 * A representatiom of the function signature suitable for display to
+	 * the user.
+	 *
+	 * @param functionName the name of the function
+	 */
+	default String getSignature(String functionName) {
+		return functionName;
+	}
+
+	/**
+	 * Determines if invocations of this function require an argument
+	 * list. (Some SQL functions, for example {@code current_date}, do
+	 * not require an argument list.
```

Comment:
If the code was not moved from elsewhere, I think a ')' is missing for the above Javadoc